app/src/main/java/org/liberty/android/fantastischmemo/ui/StatisticsScreen.java

```diff
@@ -246,6 +257,8 @@ public Chart generateChart(BarData data) {
             chart.getXAxis().setTextColor(Color.WHITE);
             chart.getAxisLeft().setTextColor(Color.WHITE);
             chart.getAxisRight().setTextColor(Color.WHITE);
+            chart.getAxisLeft().setValueFormatter(new ChartYAxisValueFormatter());
```

Comment:
I understood your meaning. I just wanted to make sure that you want to have the static member in `StatisticsScreen` class, since it sort of increase the coupling between the main and inner classes. 

I'll change this way.